libobs/obs-service.c

```diff
@@ -119,6 +119,21 @@ const char *obs_service_get_name(const obs_service_t *service)
 		service->context.name : NULL;
 }
 
+void * obs_service_query(const obs_service_t *service, int query_id, ...)
```

Comment:
A world without VA lists is a better world.

Reply:
By my understanding, this won't be needed at all with the upcoming service auth changes, so this should be going away.